packs/travis_ci/actions/get_git_token.py

```diff
@@ -0,0 +1,23 @@
+from lib.action import TravisCI
+import json
+import requests
+
+
+class GetGitToken(TravisCI):
+    def run(self):
+        username = self.config['username']
+        password = self.config['password']
+        url = 'https://api.github.com/authorizations'
```

Comment:
saw @Kami's comment on the other PR. It's fine as a constant. Also, looks like they are unversioned. https://api.github.com/